lib/simulator-xcode-6.js

```diff
@@ -162,7 +162,7 @@ class SimulatorXcode6 {
     ];
     let pv = await this.getPlatformVersion();
     if (pv !== '7.1') {
-      if (pv[0] !== '9') {
+      if (pv !== '8.0' && pv !== '8.1' && pv[0] !== '9') {
```

Comment:
with this kind of thing where we have some magic values encoded in a conditional, i'd love to see a comment explaining why we are doing something with just these platforms.

i realize the comment should already have existed so it's not your fault, but could you add one?